docs/spark/tutorials/databricks-deployment.md

```diff
@@ -100,9 +100,12 @@ Now that you have the Databricks CLI installed, you need to set up authenticatio
 
 You should now be able to access any Azure Databricks clusters you create and upload files to the DBFS.
 
-## Download worker dependencies
+## Download Microsoft Spark Worker dependencies
```

Comment:
I have reverted this change although I am not sure I understand how the previous change offends the guidelines asking to keep headings to 2 lines, there was just one line to the heading no? Perhaps I am missing something.

Reply:
The text is in one line. However, the headers on the sidebar wrap text. This is what the current one looks like. If you were to expand to Microsoft.Spark.Worker, it would go into 3+ lines.

![image](https://user-images.githubusercontent.com/46974588/99349082-142b6f00-2869-11eb-8867-74ad9a2d6434.png)